packages/css/src/dialog/README.md

```diff
@@ -0,0 +1,22 @@
+# Dialog
+
+The dialog provides feedback or reveals additional information or functionality while maintaining the context of an underlying page.
```

Comment:
```suggestion
A Dialog allows the user to focus on one task or a piece of information, by popping-up and blocking the page content until the modal task is completed, or until the user dismisses the action.
```

Reply:
I'd rather not use the term pop-up and I was happy with the phrasing of keeping the context of the page. That's why we use a semi transparent backdrop. Also it's not really blocking the page if you're able to dismiss the dialog.

Reply:
Why do you rather not use the term pop-up? We can get rid of it I think, it doesn't really add much. 

To me it doesn't really feel like the core function of the Dialog that it (only visually) maintains the context of an underlying page. The core function is taking you out of the underlying page and forcing you to take action before you can return. 

> Also it's not really blocking the page if you're able to dismiss the dialog.

Which is taking action, right? So it's blocked until the user does something.